app/src/main/java/com/hover/stax/domain/use_case/accounts/CreateAccountsUseCase.kt

```diff
@@ -1,16 +0,0 @@
-package com.hover.stax.domain.use_case.accounts
-
-import com.hover.sdk.sims.SimInfo
-import com.hover.stax.channels.Channel
-import com.hover.stax.domain.repository.AccountRepository
-
-class CreateAccountsUseCase(private val accountsRepository: AccountRepository) {
```

Comment:
This usecase doesn't do anything. Use cases should do something.

From https://developer.android.com/topic/architecture/domain-layer: "You should only use it when needed-for example, to handle complexity or favor reusability."